backend/src/service/Boards/BoardsService.js

```diff
@@ -0,0 +1,18 @@
+import { BoardsRepository } from '../../repository/Boards/Boards.repository.js';
+import BoardsValidationSchema from './BoardsValidationSchema.js';
+import { json } from 'express';
+
+export class BoardsService {
+  static async addOne(newBoard) {
+    console.log(newBoard);
+
+    const { error } = BoardsValidationSchema.validate(newBoard);
```

Comment:
A mogłabyś wyrzucić to do osobnej funkcji (poza .service.js, ale w tym samym folderze/module)?
Np. `validateBoard(newBoard)`?
Wtedy ta funkcja zajmie się tym, jak validować i tego pliku(.service.js) nie będzie obchodzić czy ta funkcja wykorzystuje do tego jakąś bibliotekę czy nie. No i bedziemy mogli tą funkcją łatwo otestować, pisząc prosty test.
Wredy zamiast nazywać plik `BoardsValidationSchema.js` nazwiemy go `validateBoard.js`, a test `validateBoard.spec.js`.

Dodatkowo "BoardsValidationSchema" nie za dużo nam mówi, a "validateBoard" już całkiem sporo :)